code-challenges.test.js

```diff
@@ -10,37 +10,192 @@
+
+  //     at Object.expect (code-challenges.test.js:27:7)
 
 // b) Create the function that makes the test pass.
+// Use .map and .split & toUpperCase.
+// Use split to split the string into seperate words and put it into a new array that'll then get uppercased. It should return the n
+
+const sentenceJoiner = (variable) => {
+  return variable.map((newPeople)=>{
+    let variable1 = newPeople.split('')
+    let variable2 = variable1.toUpperCase
+    return variable2
+
+  })
+}
+const newPeople = sentenceJoiner(people);
+console.log(newPeople)
+
+
+
+
 
-// --------------------2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 
-// a) Create a test with an expect statement using the variables provided.
 
+
+
+
+
+
+
+
+// 2) Create a function that takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
 const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 // Expected output: [ 2, 0, -1, 0 ]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+
+// a) Create a test with an expect statement using the variables provided.
+describe("arrayDivider", ()=>{
+  it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", ()=>{
+    expect(arrayDivider(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+    expect(arrayDivider(hodgepodge2)).toEqual([ 2, 1, -1 ])
+  })
+})
+
+
+
+// FAIL  ./code-challenges.test.js
+// arrayDivider
+//   ✕ takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3. (1 ms)
+
+// ● arrayDivider › takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.
+
+//   ReferenceError: arrayDivider is not defined
+
+//     85 | describe("arrayDivider", ()=>{
+//     86 |   it("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3.", ()=>{
+//   > 87 |     expect(arrayDivider(hodgepodge1)).toEqual([ 2, 0, -1, 0 ])
+//        |     ^
+//     88 |     expect(arrayDivider(hodgepodge2)).toEqual([ 2, 1, -1 ])
+//     89 |   })
+//     90 | })
+
+//     at Object.expect (code-challenges.test.js:87:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
+// Time:        0.533 s, estimated 1 s
+
 // b) Create the function that makes the test pass.
+ // Use filter to filter out anything out of the array that isn't a number & map to create a new aray out of the filtered array. With the new filtered array i'll divide each number by 3 and put it into a new array. 
+const arrayDivider = (array) => {
+  const newArray = array.filter(item =>typeof item === 'number');
+  const newArray2 = newArray.map(num => num %3 );
+  return newArray2
+};
+console.log(arrayDivider(hodgepodge1));
+console.log(arrayDivider(hodgepodge2));
+
+
+
+
```

Comment:
Fantastic work here, Valeria! You can even chain the .map onto the end of newArray and just return that result. 
Awesome job!